tests/test_io.py

```diff
@@ -6,7 +8,10 @@
 import trackintel as ti
 from trackintel.preprocessing import positionfixes
 from trackintel.preprocessing import staypoints
-
+from trackintel.preprocessing.triplegs import smoothen_triplegs
+import pandas as pd
+import numpy as np
+from matplotlib import pyplot
```

Comment:
Not sure if this should be in a test, as it usually is difficult to automatically test visual things (not impossible though I guess).